src/main/jbake/content/docs/index.ad

```diff
@@ -25,12 +25,12 @@
 There exists a manual and Javadoc API documentation for Apache OpenNLP. The manual
 explains how the various OpenNLP components can be used and trained.
 
-### Apache OpenNLP 1.8.0 documentation
+### Apache OpenNLP {opennlp_version} documentation
 
-* link:/docs/1.8.0/manual/opennlp.html[Apache OpenNLP Manual]
-* link:/docs/1.8.0/apidocs/opennlp-tools/index.html[Apache OpenNLP Tools Javadoc]
-* link:/docs/1.8.0/apidocs/opennlp-uima/index.html[Apache OpenNLP UIMA Javadoc]
-* link:/docs/1.8.0/apidocs/opennlp-brat-annotator/index.html[Apache OpenNLP BRAT Annotator Javadoc]
-* link:/docs/1.8.0/apidocs/opennlp-morfologik-addon/index.html[Apache OpenNLP Morfologik Addon Javadoc]
+* link:/docs/1.7.2/manual/opennlp.html[Apache OpenNLP Manual]
```

Comment:
Not sure, see this part of the pull request comment

>Notice the manual URL is set to 1.7.2, as we do not have the manual generated for 1.8.0 - TBH I do not know how to generate it.

Not sure why we do not have the manual for 1.8.0. When I generate the site with `mvn clean package`, I get the Javadocs for 1.8.0, but no manual...